io.asgardeo.java.oidc.sdk/src/main/java/io/asgardeo/java/oidc/sdk/HTTPSessionBasedOIDCProcessor.java

```diff
@@ -84,9 +86,15 @@ public void handleOIDCCallback(HttpServletRequest request, HttpServletResponse r
         SessionContext sessionContext = defaultOIDCManager.handleOIDCCallback(request, response, requestContext);
 
         if (sessionContext != null) {
-            clearSession(request);
-            HttpSession session = request.getSession();
-            session.setAttribute(SSOAgentConstants.SESSION_CONTEXT, sessionContext);
+            if (sessionContext.getAdditionalParams().containsKey(IS_LOGOUT) &&
```

Comment:
Wouldn't it be better to have `SSOAgentConstants.IS_LOGOUT` here?
Would remove any ambiguity in the future.